definitions/velocity.proto

```diff
@@ -42,3 +42,57 @@ message DetectMotionRequest {
   required PlatformType platform  = 6;
   required double timestamp       = 7;
 }
+
+message HandshakeRequest {
+  required string authToken       = 1;
+  optional string ifa             = 2;
+  optional string userId          = 3;
+  optional string appId           = 4;
+  optional PlatformType platform  = 5;
+}
+
+
+message HandshakeResponse {
+  optional double sampleSize      = 1;
+  optional double captureInterval = 2;
+  optional bool   canDetectMotion = 3;
+  optional bool   canLabelMotion  = 4;
+}
+
+
+message SensorsRequest {
+  required string impressionId    = 1;
+  repeated Sensor sensors         = 2;
+  optional uint32 sequenceIndex   = 3;
+  repeated string modelNames      = 4;
+  optional string ifa             = 5;
+  optional string userId          = 6;
+  optional string appId           = 7;
+  optional string metadata        = 8;
+  optional double timestamp       = 9;
+  optional PlatformType platform  = 10;
+}
+
+message ModelPrediction {
+  required string modelName = 1;
+  repeated Prediction predictions = 2;
+}
+
+message Prediction {
+  required string name = 1;
+  optional float confidence = 2;
+  optional double timestamp = 3;
+}
+
+message Error {
+  required string errorMessage = 1;
+  optional string errorCode    = 2;
+}
+
+message PredictionsResponse {
```

Comment:
for plain capture requests we will not return any predictions. Maybe more general ``RequestMessage`` and ``ResponseMessage`` would be better?